tests/components/glances/test_config_flow.py

```diff
@@ -107,12 +108,14 @@ async def test_form_already_configured(hass):
     assert result["reason"] == "already_configured"
 
 
-async def test_options(hass):
+async def test_options(hass: HomeAssistant) -> None:
     """Test options for Glances."""
     entry = MockConfigEntry(
         domain=glances.DOMAIN, data=DEMO_USER_INPUT, options={CONF_SCAN_INTERVAL: 60}
     )
     entry.add_to_hass(hass)
+    await hass.config_entries.async_setup(entry.entry_id)
+    await hass.async_block_till_done()
```

Comment:
Without these two lines, then the test was dependent on previous tests, and failed when running stand-alone.